assets/css/styles.css

```diff
@@ -807,12 +824,58 @@ input:focus + .input-label {
         padding: 8px 0 16px 0;
     }
 
-    nav > ul > li {
+    /* nav > ul > li {
         margin: 0 5px;
-    }
+    } */
 
     .project-text {
         padding: 16px 0 0 0;
     }
 }
 
+@media only screen and (max-width: 770px) {
```

Comment:
When under 770px width then hide the "ePortfolio" text, make the logo font size smaller and shorten the "Certificates" link to "Cert".